task_DEV_4/task_DEV_4/Rectangle.cs

```diff
@@ -1,40 +1,30 @@
 ﻿namespace task_DEV_4
 {
-    class Rectangle
+    class Rectangle : Square
     {
-        public int height;
-        public int width;
+        public double Height { get; set; }
+        public double Width { get; set; }        
 
-        public Rectangle()
+        public Rectangle(double height, double side) : base(side)
         {
-
-        }
-
-        public Rectangle(int height, int width)
-        {
-            this.height = height;
-            this.width = width;
-        }
-
-        public void SetRectangleParameters(int height, int width)
-        {
-            this.height = height;
-            this.width = width;
+            Height = height;
+            Width = side;
```

Comment:
There should be setting only one property.

Reply:
Done